features/aanhef.feature

```diff
@@ -649,129 +649,137 @@ Rule: Voor het bepalen van de aanhef gaat gebruik van de adellijke titel van de
+    | naam                                                               | waarde            |
+    | datum huwelijkssluiting/aangaan geregistreerd partnerschap (06.10) | <datum partner 1> |
+    En de persoon heeft een 'partner' met de volgende 'naam' gegevens
     | naam                  | waarde |
     | voorvoegsel (02.30)   |        |
     | geslachtsnaam (02.40) | Roodt  |
-    En de partner heeft de volgende aangaanHuwelijkPartnerschap gegevens
-    | naam                  | waarde   |
-    | datum aangaan (06.10) | 19610808 |
+    En de 'partner' heeft de volgende 'aangaanHuwelijkPartnerschap' gegevens
+    | naam                                                               | waarde            |
+    | datum huwelijkssluiting/aangaan geregistreerd partnerschap (06.10) | <datum partner 2> |
     Als personen wordt gezocht met de volgende parameters
     | naam                | waarde                          |
     | type                | RaadpleegMetBurgerservicenummer |
     | burgerservicenummer | 999992934                       |
     | fields              | naam.aanhef                     |
-    Dan heeft de persoon met burgerservicenummer '999992934' de volgende 'naam' gegevens
-    | naam   | waarde                     |
-    | aanhef | Geachte mevrouw Geel-Groen |
+    Dan heeft de response een persoon met de volgende 'naam' gegevens
+    | naam   | waarde                 |
+    | aanhef | Geachte mevrouw <naam> |
+
+    Voorbeelden:
+    | datum partner 1 | datum partner 2 | naam        | omschrijving                                                                                                                      |
+    | 19580401        | 19610808        | Geel-Groen  |                                                                                                                                   |
+    | 19580401        | 19610800        | Geel-Groen  |                                                                                                                                   |
+    | 19580401        | 19610000        | Geel-Groen  |                                                                                                                                   |
+    | 19580401        | 00000000        | Roodt-Groen | partner met datum onbekend wordt eerste partner                                                                                   |
+    | 19610800        | 19610808        | Geel-Groen  | jaar en maand zijn bekend en zijn gelijk, partner met dag onbekend wordt eerste partner                                           |
+    | 19610000        | 19610800        | Geel-Groen  | partner 1 alleen jaar bekend, partner 2 jaar en maand bekend, jaar gelijk, partner met maand en dag onbekend wordt eerste partner |
 
 Rule: Bij meerdere huwelijken/partnerschappen die allen ontbonden zijn, worden de naamgegevens van de laatst ontbonden partner gebruikt voor het samenstellen van de aanhef
```

Comment:
Kunnen we dit niet in de RvIG implementatie oplossen, doordat de GbaPartner alleen de laatst ontbonden partner bevat in plaats van alle ontbonden partners? Of zijn er andere features die alle ontbonden partners nodig hebben? Dan krijgt de proxy altijd maximaal 1 ontbinden partner